packages/next/client/index.tsx

```diff
@@ -138,6 +138,11 @@ if (process.env.__NEXT_I18N_SUPPORT) {
   }
 }
 
+if (process.env.__NEXT_SCRIPT_LOADER && data.scriptLoader) {
```

Comment:
This is being included in the main bundle despite being a conditional require
Any idea how to fix this @Timer @ijjk

Reply:
I don't see it being included in the main bundle in the stats [here](https://github.com/vercel/next.js/pull/22038#issuecomment-780057683), was it previously being included?